digitalocean/droplet/resource_droplet.go

```diff
@@ -663,7 +663,10 @@ func resourceDigitalOceanDropletUpdate(ctx context.Context, d *schema.ResourceDa
 			}
 		}
 		for volumeID := range leftDiff(oldIDSet, newIDSet) {
-			detachVolumeIDOnDroplet(d, volumeID, meta)
+			err := detachVolumeIDOnDroplet(d, volumeID, meta)
+			if err != nil {
+				return nil
```

Comment:
Let's return `err` instead
```suggestion
				return err
```